### DISCUSSION TITLE
autoplay

### DISCUSSION THREAD

**Discussion by michaelherger** [librespot-org/librespot#748] [General]
We now have `autoplay` in librespot and in the Spotify apps. librespot's will override the one in the app. Does this make sense? Shouldn't that option live in one place only? I can have autoplay turned off in the app, if enabled in Spotty it would continue to play anyway. But it wouldn't unless enabled in librespot, no matter the setting in the app.

**Comment by ashthespy** [comment]
This seems to be a client only feature -- we weren't able to find any traffic via spric that communicates this from client to client https://github.com/librespot-org/librespot/issues/294 
That being said, I don't think an in-depth investigation was done.. Feel free to report back if you find something! :-)

**Reply by michaelherger** [reply]
I don't have any "real" Connect hardware. I though the "autoplay" setting in the app would control whatever hardware I was playing on. But that's ok. Good to know it's a client side feature. Thanks!
